### PR TITLE
Nested categories

### PR DESCRIPTION
**Pull request by gpasztor87**

### CODE REVIEW

**Comment by keiosweb** [comment]
Nice! :+1:

**Comment by gpasztor87** [comment]
Thanks!

File: models/Post.php [hunk 1]
```diff
@@ -132,7 +136,9 @@ public function scopeListFrontEnd($query, $options)
          * Categories
          */
         if ($categories !== null) {
-            if (!is_array($categories)) $categories = [$categories];
+            $categories = Category::find($categories);
```
Comment (daftspunk) [inline models/Post.php]: No more support for multiple categories... Perhaps we should leave "categories" intact and add a new option called "category" that does this instead.

Comment (gpasztor87) [inline models/Post.php]: I fixed it.

**Comment by daftspunk** [comment]
Thanks!

**Pull request merged**